metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/EnaObjectQuery.java

```diff
@@ -25,4 +25,6 @@
     String ANALYSIS_QUERY = "SELECT ANALYSIS_XML FROM ERA.ANALYSIS WHERE ANALYSIS_ID = :accession";
 
     String SAMPLE_QUERY = "SELECT SAMPLE_XML FROM ERA.SAMPLE WHERE SAMPLE_ID = :accession";
+
+    String SAMPLE_QUERY_LIST = "SELECT SAMPLE_XML FROM ERA.SAMPLE WHERE SAMPLE_ID IN(:accession)";
```

Comment:
```suggestion
    String SAMPLE_QUERY = "SELECT SAMPLE_XML FROM ERA.SAMPLE WHERE SAMPLE_ID IN(:accessions)";
```

Reply:
SAMPLE_QUERY is already present above.
Once this approach to speed-up is accepted, i can use SAMPLE_QUERY if there is no dependency.